oioioi/sinolpack/package.py

```diff
@@ -1139,9 +1139,11 @@ def _prepare_overrides(self, lang):
         """
         tests = Test.objects.filter(problem_instance=self.main_problem_instance)
         for test in tests:
-            LanguageOverrideForTest.objects.create(
-                time_limit=test.time_limit,
-                memory_limit=test.memory_limit,
+            LanguageOverrideForTest.objects.update_or_create(
+                defaults={
+                    'time_limit': test.time_limit,
+                    'memory_limit': test.memory_limit,
+                },
                 test=test,
                 language=lang,
             )
```

Comment:
This duplicates #228, which also covers the bug in a test.

Reply:
I see, it's a pity two people had to come up with solution to the same problem.
I would be happy to include your tests with this change. Are You fine with that?

Reply:
Yeah, it's fine.